stock_free_quantity/models/product_template.py

```diff
@@ -10,7 +10,7 @@ def _compute_quantities_dict(self):
         prod_available = super()._compute_quantities_dict()
         for template in self:
             free_qty = 0
-            for p in template.with_context(active_test=False).product_variant_ids:
```

Comment:
@florian-dacosta as original author, do you have more insights in the need to include archived variants here?

Reply:
@florian-dacosta ?

Reply:
Hello,
Sorry, I missed the first ping.

I don't remember no. 
I think it does not change a lot because archived product should not have stock level equal to 0.
But anyway, ok for the change on my side